test/FunctionsV2/TestCleanup.cs

```diff
@@ -0,0 +1,94 @@
+﻿// Copyright (c) .NET Foundation. All rights reserved.
+// Licensed under the MIT License. See LICENSE in the project root for license information.
+
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Threading.Tasks;
+using DurableTask.AzureStorage;
+using FluentAssertions.Common;
+using Microsoft.Azure.WebJobs.Extensions.DurableTask.Tests;
+using Microsoft.Azure.WebJobs.Host.TestCommon;
+using Microsoft.WindowsAzure.Storage;
+using Microsoft.WindowsAzure.Storage.Blob;
+using Xunit;
+using Xunit.Abstractions;
+
+namespace WebJobs.Extensions.DurableTask.Tests.V2
+{
+    public class TestCleanup
+    {
+        private readonly ITestOutputHelper output;
+
+        private readonly TestLoggerProvider loggerProvider;
+
+        public TestCleanup(ITestOutputHelper output)
+        {
+            this.output = output;
+            this.loggerProvider = new TestLoggerProvider(output);
+        }
+
+        // This test should never fail. The sole purpose is to cleanup old taskhubs in the CI
+        // storage account to prevent clutter now that TaskHub names are non-deterministic.
+        [Fact]
+        [Trait("Category", PlatformSpecificHelpers.TestCategory)]
+        public async Task CleanupOldAzureStorageTaskHubs()
```

Comment:
This does seem like a good addition. And I see that you're using AzureStorageOrchestratonService to delete each task hub. Should we add this logic to AzureStorageOrchestrationService to clean up all old task hubs?

Reply:
An AzureStorageOrchestrationService is specifically tied to one task hub by definition, so I don't feel comfortable adding logic to delete other task hubs on the storage account.